src/esp/gfx/PbrShader.h

```diff
@@ -10,6 +10,7 @@
 #include <Corrade/Containers/ArrayView.h>
 #include <Corrade/Containers/EnumSet.h>
 #include <Magnum/GL/AbstractShaderProgram.h>
+#include <Magnum/GL/CubeMapTexture.h>
```

Comment:
FYI, for reducing compile times you can do the include just in the cpp file (and then of course in files that actually use it, but it's there already anyway), here a forward declaration is enough.

Reply:
Yeah, good catch. I did not pay attention to it. will change.